backend/src/common/enums/file/resource-type.enum.ts

```diff
@@ -0,0 +1,8 @@
+enum ResourceType {
+  auto = 'auto',
+  image = 'image',
+  video = 'video',
+  raw = 'raw',
```

Comment:
```suggestion
  AUTO = 'auto',
  IMAGE = 'image',
  VIDEO = 'video',
  RAW = 'raw',
```
Enum should have keys as constants